gulp/tasks/serve.js

```diff
@@ -1,6 +1,17 @@
 'use strict';
 
 const
-    gulp = require('gulp');
+    gulp = require('gulp'),
+    nodemon = require('gulp-nodemon');
 
-gulp.task('serve', ['resume', 'watch']);
\ No newline at end of file
+
+gulp.task('serve', ['resume','watch'],function () {
+  //initiate server files and auto restart server whenever changes are made
+
+  ///*Note change ext to watch for hbs files
+  nodemon({
```

Comment:
I'm still wondering if this nodemon even does anything for us anymore. If you don't have it listen to handlebars changes, it doesn't really do anything.